elytron/WFCORE-5145-ssl-allowed-protocols.adoc

```diff
@@ -0,0 +1,111 @@
+= [WFCORE-5145] Elytron server-ssl-context allowed protocols
+:author:            Sonia Zaldana Calles
+:email:             szaldana@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Elytron currently supports 6 SSL/TLS protocols. Namely, SSLv2, SSLv3, TLSv1, TLSv1.1, TLSv1.2, and
+TLSv1.3. However, older JDK versions use ``SSLv2Hello`` during the initial
+SSL handshake message where the SSL version that will be used for the rest of the handshake is negotiated.
+Newer JDK versions disable this protocol by default, but do provide the ability to
+re-enable it if needed. This task is to ensure Elytron can make use of the ``SSLv2Hello`` protocol to ensure feature parity with legacy security.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFCORE-5145[WFCORE-5145]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-13916[WFLY-13916]
+* https://issues.redhat.com/browse/ELY-2024[ELY-2024]
+* https://issues.jboss.org/browse/EAP7-1542[EAP7-1542]
+
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+TBD
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
```

Comment:
@SoniaZaldana Since this was tested by Engineering, this section can now be updated. Thanks.